migrations/changesets/changeset-1-CreateContragent.xml

```diff
@@ -0,0 +1,33 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+    <changeSet id="1" author="dbobro">
+	<preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists schemaName="public" tableName="contragent" />
```

Comment:
Could you please explain what do you suppose to do with such table?

Reply:
changed.